models/commons/src/address/address-draft/presets/index.ts

```diff
@@ -1,6 +1,7 @@
+import changeHistoryData from './change-history-data';
 import empty from './empty';
 import sampleDataFashion from './sample-data-fashion';
 
-const presets = { empty, sampleDataFashion };
+const presets = { changeHistoryData, empty, sampleDataFashion };
```

Comment:
One thing that we did was to store all our presets in the `sample-data-fashion` directory to clearly separate them from any other teams' presets. Would that strategy be appropriate for you here?

Reply:
Lol, I spoke too soon. In fact, I can't read